packages/dialog/src/react/index.js

```diff
@@ -87,32 +90,29 @@ class Dialog extends React.Component {
     this.handleKeyUp = this.handleKeyUp.bind(this)
   }
 
-  componentDidMount() {
-    if (this.el && !this.props.disableFocusOnMount) this.el.focus()
```

Comment:
What was your reasoning in getting rid of this?  I can't see where its function is handled elsewhere.

Reply:
i will update the `FocusLock` to make it's self focusable if no focusable children

Reply:
Dialog(that aren't modal/onClose) should also focus on mount

Reply:
update FocusLock to have active/inactive or locked/unlocked. rename FocusManager? trapped(bool?)